docs/Chipyard-Basics/Initial-Repo-Setup.rst

```diff
@@ -62,27 +62,25 @@ Start by checking out the proper Chipyard version. Run:
     # note: this may not be the latest release if the documentation version != "stable"
     git checkout |version|
 
-Next run the following script to create Chipyard's Conda environment including a pre-built RISC-V toolchain.
+Next run the following script to setup Chipyard completely with a specific toolchain.
```

Comment:
We should provide some warning that the default "full" install will take a long time, and that the user should make sure this step doesn't get cancelled halfway though (don't run over ssh without tmux).